api-docs/operator-intro.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Operator Registry Introduction
+category: 62e8eba5923a5105f57589e7
+---
+Our Operator API provides a list of APIs enabling management of operators.
+
+The document is intended for developers, who wants to integrate systems, write client libraries or other interactions on an API-level.
+
+## Interface
+
+The API expose an OpenAPI JSON REST interface that you can call directly using your OAuth 2.0 access token. See [Access IRIS APIs](../reference/access-token) for further details on how to obtain an access token.
+
+## Concepts
+
+The operator registry is built on the following concepts
+
+- An **asset** is a trackable piece of equipment e.g. a machine or an accessory.
+- An **operator** is a user that operates assets. Operators can sign in to the Trackunit Go app if invited.
+
```

Comment:
Since we have asset here, should we include Digital-key as a concept?

Reply:
I added assets to explain the operator term. I will leave the keys out for now since they don't contribute to explain what an operator is.